command/config/generate/types.go

```diff
@@ -2,17 +2,12 @@ package generate
 
 // DSConfig is the struct for .deepsource.toml file
 type Analyzer struct {
-	Name                string       `toml:"name,omitempty" json:"name,omitempty"`
-	RuntimeVersion      string       `toml:"runtime_version,omitempty" json:"runtime_version,omitempty"`
-	Enabled             bool         `toml:"enabled" json:"enabled"`
-	DependencyFilePaths []string     `toml:"dependency_file_paths,omitempty" json:"dependency_file_paths,omitempty"`
-	Meta                AnalyzerMeta `toml:"meta,omitempty" json:"meta,omitempty"`
-	Thresholds          interface{}  `toml:"thresholds,omitempty" json:"thresholds,omitempty"`
-}
-
-type AnalyzerMeta struct {
-	ImportRoot  string `toml:"import_root,omitempty" json:"import_root,omitempty"`
-	JavaVersion string `toml:"java_version,omitempty" json:"java_version,omitempty"`
+	Name                string      `toml:"name,omitempty" json:"name,omitempty"`
+	RuntimeVersion      string      `toml:"runtime_version,omitempty" json:"runtime_version,omitempty"`
+	Enabled             bool        `toml:"enabled" json:"enabled"`
+	DependencyFilePaths []string    `toml:"dependency_file_paths,omitempty" json:"dependency_file_paths,omitempty"`
+	Meta                interface{} `toml:"meta,omitempty" json:"meta,omitempty"`
```

Comment:
Do we need this to be an interface?

Reply:
Yes, since we don't know the exact structure of analyzer meta. Different analyzers have meta whose number of fields and their type varies.